### DISCUSSION TITLE
Add Support For Perplexica

### DISCUSSION THREAD

**Discussion by Iory1998** [heshengtao/comfyui_LLM_party#90] [Ideas]
Hi!

As I mention in the title, I highly request support for Perplexica. It's an open alternative for Perplexity.
https://github.com/ItzCrazyKns/Perplexica

Thank you.

**Comment by heshengtao** [comment]
Added the searxng tool, which can aggregate search the whole network. Perplexica also relies on this aggregate search tool, which is equivalent to building a Perplexica in a party.

**Reply by Iory1998** [reply]
Oh, really? That's great.
Could you please prepare a few tutorials on your YouTube channel?

**Comment by heshengtao** [comment]
No problem, I will update more on YouTube in the future.
